draft-packages/likert-scale-legacy/docs/README.mdx

```diff
@@ -30,6 +30,9 @@ We typically use a 5-point Likert scale for employees to answer survey questions
 
 Our people scientists can provide case-by-case guidance for when and if a scale with more than 5 options may be appropriate.
 
+[Focus indicator](/guidelines/interaction-states)
```

Comment:
The rest of the "To Keep In Mind" sections have h3 headings, I wonder if we should do the same here for consistency.

```suggestion
### [Focus indicator](/guidelines/interaction-states)
```